packages/react-core/__tests__/operations/histogram.test.js

```diff
@@ -18,87 +19,91 @@ const buildInvalidFeatures = (columnName) => [
 ];
 
 const COLUMN = 'test';
+const COLUMN_2 = 'test_2';
 const ticks = Array.from(new Set(VALUES));
 
 describe('histogram', () => {
   test('should return an empty array due to empty features array', () => {
-    expect(histogram([])).toEqual([]);
+    expect(histogram({ data: [] })).toEqual([]);
   });
 
   describe('valid features', () => {
     test('should return an empty array due to invalid operation', () => {
-      expect(histogram(buildValidFeatures(COLUMN), COLUMN, ticks, 'pow')).toEqual([]);
+      expect(
+        histogram({
+          data: buildValidFeatures(COLUMN),
+          valuesColumns: [COLUMN],
+          ticks,
+          // @ts-ignore
+          operation: 'pow'
+        })
+      ).toEqual([]);
     });
 
     test('should return an array of bins with same length as ticks plus two, due to the addition of extreme values', () => {
-      const h = histogram(
-        buildValidFeatures(COLUMN),
-        COLUMN,
+      const h = histogram({
+        data: buildValidFeatures(COLUMN),
+        valuesColumns: [COLUMN],
         ticks,
-        AggregationTypes.COUNT
-      );
+        operation: AggregationTypes.COUNT
+      });
       expect(h.length).toBe(1 + ticks.length);
     });
 
-    test(AggregationTypes.COUNT, () => {
-      const h = histogram(
-        buildValidFeatures(COLUMN),
-        COLUMN,
-        ticks,
-        AggregationTypes.COUNT
-      );
-      expect(h).toEqual([0, 1, 2, 3, 2, 1]);
-    });
+    const RESULTS = [
+      [0, 1, 2, 3, 2, 1],
+      [0, 1, 2, 3, 4, 5],
+      [0, 1, 2, 3, 4, 5],
+      [0, 1, 2, 3, 4, 5],
+      [0, 1, 4, 9, 8, 5]
+    ];
 
-    test(AggregationTypes.AVG, () => {
-      const h = histogram(
-        buildValidFeatures(COLUMN),
-        COLUMN,
-        ticks,
-        AggregationTypes.AVG
-      );
-      expect(h).toEqual([0, 1, 2, 3, 4, 5]);
+    describe('one valuesColumns', () => {
+      Object.values(AggregationTypes).forEach((operation, idx) => {
+        test(operation, () => {
+          const groups = histogram({
+            data: buildValidFeatures(COLUMN),
+            valuesColumns: [COLUMN],
+            ticks,
+            operation
+          });
+          expect(groups).toEqual(RESULTS[idx]);
```

Comment:
Same comment here, would be clearer and more robust with `RESULTS` as an object.